code/modules/goonchat/browserOutput.css

```
@@ -308,10 +308,10 @@ h1.alert, h2.alert		{color: #99aab5;}
 .highdanger				{color: #f44747;	font-weight: bold; font-size: 1.5em;}
 .xenohighdanger			{color: #6a9955; 	font-weight: bold; font-size: 1.5em;}
 .xenoannounce           {color: #4EC9B0;    font-family: book-antiqua; font-weight: bold; font-style: italic; font-size: 1.5em;}
-.hivemind				{color: #c586c0;}
 .xenoqueen				{font-size: 1.4em;}
 .xenoshrike				{font-size: 1.2em;}
-
+.hivemind {color: #c586c0; font-size: 1.1em;}
```

```suggestion
.hivemind				{color: #c586c0; font-size: 1.1em;}
```

Gotta keep that indenting